std_msgs/Header header
std_msgs/Int32 n
ros_object_detector/DetectedObject[] objects
